amqp-postgres/dev-requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common==master
+git+https://github.com/cloudify-cosmo/cloudify-common@CY-1244-support-upload-plugin-in-cluster#egg=cloudify-common==master
```

Comment:
this file doesn't really matter and won't even really use that, however what does matter is mgmtworker/requirements.txt - if you want CI to build the mgmtworker rpm using that common branch, change it there

Reply:
You mean cloudify-manager/packaging/mgmtworker/requirements.txt, no?

Reply:
yes ofc